crates/llm-base/src/inference_session.rs

```diff
@@ -144,7 +144,7 @@ impl InferenceSession {
         };
 
         if use_gpu {
-            ggml::accelerator::initialize(0);
+            ggml::accelerator::initialize(config.cuda_device as i32, config.gpu_split);
```

Comment:
Does this work with multiple gpus? I always thought we need to define every device id we want to use.